packages/wallet-lib/src/types/Storage/methods/configure.js

```diff
@@ -13,6 +17,18 @@ module.exports = async function configure(opts = {}) {
   this.autosave = has(opts, 'autosave') ? opts.autosave : this.autosave;
   this.adapter = await configureAdapter((opts.adapter) ? opts.adapter : await getDefaultAdapter());
 
+  const version = await this.adapter.getItem('version');
+  const wallets = await this.adapter.getItem('wallets');
+  const chains = await this.adapter.getItem('chains');
+
+  if ((wallets || chains) && version !== CURRENT_VERSION) {
```

Comment:
And I guess you don't need to check `(wallet || chains)` anymore because now you are writing `version` into the store.